code/modules/mob/observer/ghost/ghost.dm

```diff
@@ -629,6 +629,11 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 			if (!M.client && (length(GLOB.clients) >= 30))
 				scps += M
 
+		for (var/scp457 in GLOB.scp457s)
+			var/mob/M = scp457
+			if (!M.client)
+				scps += M
+
```

Comment:
You ought to add it below, too, where the comment indicates or otherwise it have a join cooldown or nothing and, you know, that kinda will suck

Reply:
I don't understand what you mean, but the joining has been tested and it indeed has a join cooldown AND it indeed does work.

Reply:
Ah, yeah, I just noticed code below (picrel) 
![image](https://user-images.githubusercontent.com/41126266/216349289-bdb0b502-d526-4864-81fc-13a29ce1a2aa.png)
does indeed have an `else` for "You can't spawn for X more minutes.", my bad

Reply:
which works for all, got confused by the special snowflake cases for 049 + 06